email/email.go

```diff
@@ -11,27 +11,32 @@ import (
 
 // Client Class containing the needed methods to send an email
 type Client struct {
-	apiKey string
+	sendgridClient *sendgrid.Client
 }
 
 // NewClient class constructor function
 func NewClient(apiKey string) (*Client, error) {
 	if len(apiKey) <= 0 {
 		return nil, errors.New("Api Key should not be empty")
 	}
-	return &Client{apiKey}, nil
+
+	client := sendgrid.NewSendClient(apiKey)
+
+	return &Client{client}, nil
 }
 
 // Send an email and return an error if it fails
 func (c *Client) Send(from, to, subject, body string) error {
 	fromMail := mail.NewEmail("", from)
 	toMail := mail.NewEmail("", to)
 	message := mail.NewSingleEmail(fromMail, subject, toMail, body, body)
-	client := sendgrid.NewSendClient(c.apiKey)
+	client := c.sendgridClient
```

Comment:
```suggestion
```

Reply:
Wow, nice feature this code suggestion! But i would have to make a commit for any suggestion, so i put them together in a single commit and push in the old fashion way.